lib/active_job/queue_adapters/amazon_sqs_adapter.rb

```diff
@@ -11,16 +11,16 @@ def enqueue(job)
         _enqueue(job)
       end
 
-      def enqueue_at(job, timestamp, opts={})
+      def enqueue_at(job, timestamp)
         delay = (timestamp - Time.now.to_f).floor
         raise ArgumentError, 'Unable to queue a job with a delay great than 15 minutes' if delay > 15.minutes
-        _enqueue(job, delay_seconds: delay)
+        _enqueue(job, {}, delay_seconds: delay)
       end
 
       private
 
-      def _enqueue(job, send_message_opts = {})
-        body = job.serialize
+      def _enqueue(job, body = nil, send_message_opts = {})
```

Comment:
Maybe body can be moved into the opts here and deleted? saves us having to set it to nil elsewhere when we dont want this.  and then just:
```
body ||= send_message_opts.delete(:body) || job.serialize
```

Reply:
I think it's fine to pass nil here because it's a private method and only need to do it once. It's easier to understand this way. If I were to put it in send_message_opts, we should just require it to be json dumped by the caller too.